doc/source/shared/abbreviations.rst

```diff
@@ -111,7 +111,9 @@
 .. |T| raw:: html
 
    <span class="green">✓</span>
+.. |MELA| raw:: html
 
+   <a href="https://github.com/vbertone/mela"><abbr title="a Mellin Evolution LibrAry">
```

Comment:
you did not look at the page ... else you would have seen this:
![grafik](https://user-images.githubusercontent.com/6421325/230043619-e421db9f-81ed-49cd-9299-c160918663f7.png)
which very likely is not what you want ...
```suggestion

.. |MELA| raw:: html

   <a href="https://github.com/vbertone/mela"><abbr title="a Mellin Evolution LibrAry">MELA</abbr></a>
```

Reply:
I made the changes you suggested was unable to run it because the poe docs was running giving errors, but I have fixed it now.

Reply:
Then we need to fix these errors! what is the error? did you install the necessary packages? there is a poetry group or an python extra - again look to `pyproject.toml`

Reply:
No, nothing to fix, I was missing pandoc, its working now.

Reply:
Are you saying we have an undeclared dependency on pandoc? coming from where?

Reply:
> Are you saying we have an undeclared dependency on pandoc? coming from where?

It is used by `nbsphinx` https://nbsphinx.readthedocs.io/en/0.9.1/installation.html#pandoc

Reply:
If we find a better alternative to Jupyter Notebooks, we could consider dropping them (a better alternative -> code that is easy to run, not just written there, and possibly interleaved with text).

Maybe we can think to resort to doctests, we're not using pictures anyhow, nor tables or any other kind of fancy output, so code and text should be supported enough by Sphinx itself.

Reply:
> It is used by `nbsphinx` https://nbsphinx.readthedocs.io/en/0.9.1/installation.html#pandoc

Fine, but `nbsphinx` is in the `docs` group ... as far as I understand `pip` is not sufficient, but `apt` is needed ...

Reply:
There is already a dedicated issue for this discussion #120